lib/primitives/keyring.js

```
@@ -158,28 +166,27 @@ class KeyRing extends bio.Struct {
    * @returns {KeyRing}
    */
 
-  static fromPublic(key) {
-    return new this().fromPublic(key);
+  static fromPublic(publicKey) {
+    return new this().fromPublic(publicKey);
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)